packages/common/yarn.lock

```diff
@@ -0,0 +1,958 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Not linked to this file but is `packages/common/src/wmts/.gitkeep` needed? Or we can add the folder when we have something to put in it, no?

Reply:
not needed.  I removed it.